tests/dummy.c

```diff
@@ -81,6 +81,13 @@ struct bpf_map_def SEC("maps/dummy_array_custom") dummy_array_custom = {
 	.pinning = PIN_CUSTOM_NS,
 };
 
+struct bpf_map_def SEC("maps/dummy_sockmap") sock_map = {
+	.type = BPF_MAP_TYPE_SOCKMAP,
+	.key_size = sizeof(int),
```

Comment:
Does it worth giving a more realistic key struct? Such as:
```
struct sock_key {
        __u32 sip4;
        __u32 dip4;
        __u32 sport;
        __u32 dport;
};
```

More complex example on:
https://github.com/cilium/cilium/blob/506bd14/bpf/sockops/bpf_sockops.h#L21-L46

Reply:
agreed, have added your example